packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testSourceCodeWhenClassDoesNotExistAnymore.st

```diff
@@ -0,0 +1,23 @@
+as yet unclassified
+testSourceCodeWhenClassDoesNotExistAnymore
+
+	| messageTallyMock compiledMethod tree source |
+	
+	compiledMethod := (DummyClass>>#dummyMethodToProfile).
+	
+	messageTallyMock := MessageTallyMock new.
+	messageTallyMock method: compiledMethod.
+	 
+	tree := IPTTreeModelMock new.
+	tree currentSelection: messageTallyMock.
+		
+	DummyClass removeFromSystem.
+	source := tree selectedMessage.
+	
+	ClassBuilder new superclass: Object subclass: #DummyClass instanceVariableNames: '' classVariableNames: '' poolDictionaries: '' category: 'InteractiveProfilingTool-Tests'.
+	DummyClass compile: 'dummyMethodToProfile
+   	^ 1'.
```

Comment:
These three lines are not very readable. Please refactor.

Reply:
Do you have any suggestions ? So far I understand this is the standard way to create a new class in Squeak.

Reply:
I would move the lines in question to a function with a sensible name. Also the line is much too long. We can use newlines to break up the statement over multiple lines like this:
```smalltalk
ClassBuilder new 
    superclass: Object 
    subclass: #DummyClass 
    instanceVariableNames: '' 
    classVariableNames: '' 
    poolDictionaries: '' 
    category: 'InteractiveProfilingTool-Tests'
```